src/components/WebGLView/TrailVisualization.ts

```diff
@@ -1,11 +1,13 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
-import THREE = require('three');
+import * as THREE from 'three';
 
 // @ts-ignore
 import WhiteSq from '../../../textures/sprites/square_white.png';
 
-const fragmentShader = require('../../shaders/trail_fragment.glsl');
-const vertexShader = require('../../shaders/trail_vertex.glsl');
+// @ts-ignore
+import fragmentShader from '../../shaders/trail_fragment.glsl?raw';
```

Comment:
@dvmoritzschoefl The ?raw means that this file will be loaded as source-file directly (no url). 

Add to new config as well: https://webpack.js.org/guides/asset-modules/#replacing-inline-loader-syntax